lib/updatehub/firmware.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __FIRMWARE_H__
+#define __FIRMWARE_H__
+
+#include <flash.h>
+#include <dfu/mcuboot.h>
+#include <dfu/flash_img.h>
+
+static inline bool updatehub_get_firmware_version(char *version)
```

Comment:
Same question, why is this in a header?

Reply:
Same here

Reply:
Again, put a prototype here, and put the code in a C file.